chainio/clients/avsregistry/reader.go

```diff
@@ -355,6 +355,31 @@ func (r *ChainReader) GetOperatorFromId(
 	return operatorAddress, nil
 }
 
+func (r *ChainReader) QueryRegistrationDetail(
+	opts *bind.CallOpts,
+	operatorAddress common.Address,
+) ([]bool, error) {
+	operatorId, err := r.GetOperatorId(opts, operatorAddress)
+	if err != nil {
+		return nil, err
+	}
+	value, err := r.registryCoordinator.GetCurrentQuorumBitmap(opts, operatorId)
+	if err != nil {
+		return nil, err
+	}
+	numBits := value.BitLen()
+	quorums := make([]bool, numBits)
```

Comment:
How are you using this quorums array? Wouldn't you rather have a map instead?

Reply:
Hi, initially i thought we could use index of the array as the quorum index, but yes i have changed it to use a map for  more easier to understand

Reply:
Think you should only be inputting the non zero quorums into the map? Aka treating it as a set instead of a map. That way you would iterate only on quorums that are nonzero.

Reply:
I think it is better to show zero quorums as well? so one use this function can easily get:
1. which quorum(s) the operator is in already
2. which quorum(s) the operator is not in
so he can adjust the strategy accordingly, i.e. re-join the missing quorum again

Reply:
I see. In that case don’t think map is needed and we can revert to array then haha?

Reply:
ok

Reply:
reverted to use array